File: src/main/java/tn/esprit/testerreur/entity/Comment.java
package tn.esprit.testerreur.entity;

import jakarta.persistence.*;
import lombok.*;

import java.util.Date;
import java.util.Set;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Comment {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idComment;
    private Long idUser ;
    private String reponse ;
    private Date dateCreationCom ;
}

File: src/app/Entity/Comment.ts
import { Error } from './Error'
import {User} from "../../app/Entity/User";
export class Comment {

    idComment!: number;
    idUser?: number;
    reponse!: string;
    likes!:number;
    dateCreationCom: Date= new Date();
    error: Error = new Error();
    commentowner:User=new User;
    isOwner: boolean = false;
    constructor() {
      this.dateCreationCom = new Date();

    }
  }
